platform/components/CustomNavbar/index.tsx

```diff
@@ -0,0 +1,26 @@
+                </svg>
+            </div>
+            <div className={styles.navLinkContainer}>
+                <a className={styles.navLink} href="#">About Us</a>
```

Comment:
Class name should include only tailwind class names. Could u update those class names?